packages/netlify-cms-proxy-server/webpack.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   entry: { index: path.join('src', 'index.ts'), middlewares: path.join('src', 'middlewares.ts') },
   mode: NODE_ENV,
   target: 'node',
-  devtool: 'source-map',
+  devtool: 'cheap-module-source-map',
```

Comment:
Hi @dagda1, can you explain the motivation for this change?

Based on https://webpack.js.org/configuration/devtool/#devtool we might want to have the original high quality source maps in our production builds.